lib/model/migrations/20210903-01-backfill-encrypted-client-audits.js

```diff
@@ -11,15 +11,15 @@ const up = (db) => db.raw(`
 update submission_attachments sa set "isClientAudit"=true
   from submission_defs sd
   where
-    sa.name = 'audit.csv' and
+    sa.name = 'audit.csv.enc' and
```

Comment:
I thought about writing a new migration, but instead opted to modify the existing one. Mostly that seems cleaner to me, but I may need to manually rerun this migration on the QA server.